grpc-API.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+option csharp_namespace = "grpcAPI.Protos";
+
+service QueryAPI {
+
+    rpc GetTermItem (GetTermItemRequest) returns (TermItem);
+    rpc GetAllTermItems (GetAllTermItemsRequest) returns (stream TermItems);
+
+    rpc AddTermItem (AddTermItemRequest) returns (TermItem);
+    rpc UpdateTermItem (UpdateTermItemRequest) returns (TermItem);
+    rpc DeleteTermItem (DeleteTermItemRequest) returns (DeleteTermItem);
+
+    rpc InsertTermItem (stream TermItem) returns (InsertTermItem);
```

Comment:
You don't need both an Insert and an Add, they do the same thing. Remove this one